site/content/integrate/apps/structure/bindings/embedded/_index.md

```diff
@@ -0,0 +1,125 @@
+| `location`                                                                        | string                                              | Option name. The whole location path will be provided in the context.                                                                                  |
+| `label`                                                                           | string                                              | User-facing string. Defaults to location. Must be unique in its level.                                                                                 |
+| `submit`{{<compass-icon icon-star "At least one of submit or form is required">}} | [Call]({{<ref "/integrate/apps/structure/call">}})  | (_Optional_) Call to perform when the option is selected. You must provide a call if there is no form, or the form itself does not have a call.        |
+| `form`{{<compass-icon icon-star "At least one of submit or form is required">}}   | [Form]({{<ref "/integrate/apps/structure/forms">}}) | (_Optional_) Form to open in a modal form when the option is selected. You must provide a Form with a Call if there is no Call defined in the Binding. |
+
+{{<note "Note:" icon-star "At least one of submit or form is required">}}
+At least one of the `submit` or `form` fields must be specified for each embedded select option.
+{{</note>}}
+
+The following [Golang driver]({{<ref "/integrate/apps/drivers/golang">}}) example shows an embedded binding that contains a select field:
+
+```go
+post := &models.Post{
+    ChannelID: callRequest.Context.Channel.ID,
+}
+// NOTE: apps.PropAppBindings == "app_bindings"
+post.AddProp(apps.PropAppBindings, []apps.Binding{
+    Location:    "embedded",
+    AppID:       appManifest.AppID,
+    Description: "Select your favourite coffee roast",
+    Bindings: []apps.Binding{
+        Location: "coffee-roast",
+        Label:    "Coffee roast",
+        Bindings: []apps.Binding{
+            {
+                Location: "dark-roast",
+                Label:    "Dark roast",
+                Submit:   apps.NewCall("/submit-coffee-roast"),
+            },
+            {
+                Location: "medium-roast",
+                Label:    "Medium roast",
+                Submit:   apps.NewCall("/submit-coffee-roast"),
+            },
+            {
+                Location: "light-roast",
+                Label:    "Light roast",
+                Submit:   apps.NewCall("/submit-coffee-roast"),
+            },
+        },
+    },
+})
+client := appclient.AsBot(callRequest.Context)
+created_post, err := client.CreatePost(post)
+if err != nil {
+    // handle the error
+}
+```
+
+![Screenshot of embedded select example](embedded-binding-select-example.png)
```

Comment:
Can we have an example of what the `Call` payload looks like when the select binding is submitted?

Reply:
Great idea! I've added an example. :)